packages/xsnap/test/test-boot-lockdown.js

```diff
@@ -3,7 +3,7 @@ import * as childProcess from 'child_process';
 import * as os from 'os';
 import * as fs from 'fs';
 import * as path from 'path';
-import { xsnap } from '@agoric/xsnap';
```

Comment:
I hear that the latest Node.js allows self-referential packages specifically to allow this pattern. Can’t wait.